src/main/java/net/sf/jabref/gui/ParserResultWarningDialog.java

```diff
@@ -0,0 +1,85 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.gui;
+
+import java.util.ArrayList;
+
+import javax.swing.JOptionPane;
+
+import net.sf.jabref.importer.ParserResult;
+import net.sf.jabref.logic.l10n.Localization;
+
+public class ParserResultWarningDialog {
+
+    private final ParserResult parserResult;
+    private final int maxWarnings;
+    private final JabRefFrame jabRefFrame;
+    private final int dbCount;
```

Comment:
Database number (which tab to activate when showing the warning dialog).

I'll fix all things later on and comment where appropriate. Out of laptop battery now. :-)

Thanks for the feedback!